puppet/modules/quickstack/manifests/pacemaker/ceilometer.pp

```diff
@@ -133,9 +133,9 @@
       "openstack-ceilometer-alarm-evaluator",
       "openstack-ceilometer-alarm-notifier",
       "openstack-ceilometer-notification"]:
-      clone_opts      => "interleave=true",
-      resource_params => 'start-delay=10s',
       #monitor_params  => {'start-delay'     => '10s'},
+      resource_params => "clone interleave=true",
+      operation_opts  => "monitor start-delay=10s",
```

Comment:
Getting ready to test this - there was discussion on the BZ of cleaning up the start-delays, as the are typically a sign of a problem according to Andrew.  Have you tried this at all, and do we want to include dropping delays as part of this BZ?  Also, to clarify in the code being changed here, there are some cases where 'monitor was previously used, but in current release, we were deliberately NOT using monitor delays.  The remaining delays were in fact meant to be a delay of starting the service, so some of these changes appears inconsistent with that (though I acknowledge, based on the discussion in the BZ, that perhaps even these were not actually behaving as intended).  It seems to me that nobody thinks ANY of the delays are needed anymore, and this may be a very good time to do away with them. Thoughts?